code/modules/clothing/spacesuits/void/station.dm

```diff
@@ -316,11 +316,28 @@
 	light_overlay = "helmet_light_dual"
 
 /obj/item/clothing/head/space/void/science
-    var/list/icon_states = list("moebiushelmb","moebiushelmr", "moebiushelmp","moebiushelmg", "moebiushelmy", "moebiushelmw") //TODO: a manual selection anytime.
+	var/list/icon_states = list(
+		"Blue" = "moebiushelmb",
+		"Red" = "moebiushelmr",
+		"Purple" = "moebiushelmp",
+		"Green" = "moebiushelmg",
+		"Yellow" = "moebiushelmy",
+		"White" = "moebiushelmw")
+
+/obj/item/clothing/head/space/void/science/verb/recolor()
+	set name = "Change helmet's color"
+	set category = "Object"
+	set src in usr
+
+	var/color = input(usr, "Colors available", "Visor configuration", "moebiushelmb") as null|anything in icon_states
```

Comment:
```suggestion
	var/color = input(usr, "Available colors", "Visor configuration", "moebiushelmb") as null|anything in icon_states
```
And add a check if(color) to avoid null case